src/NuGet.Core/NuGet.Configuration/Resources.resx

```diff
@@ -221,6 +227,12 @@
   <data name="TextShouldNotBeEmpty" xml:space="preserve">
     <value>Text elements should not be empty.</value>
   </data>
+  <data name="TrustedSignerMustHaveCertificates" xml:space="preserve">
+    <value>A trusted signer entry must have at least one certificate entry.</value>
+  </data>
+  <data name="UnknownHashAlgorithmNotSupported" xml:space="preserve">
+    <value>Unknown hash algorithm is not supported.</value>
```

Comment:
Would it be helpful if we call out which value that is?

Reply:
This is only thrown when the item is created in memory (basically the user passed `HashAlgorithm.Unknown` to the method), when parsed we do say which value is the invalid one/